mne/decoding/tests/test_ems.py

```diff
@@ -53,4 +54,34 @@ def test_ems():
     assert_equal(n_expected, len(surrogates))
     assert_equal(n_expected, len(conditions))
     assert_equal(list(set(conditions)), [2, 3])
+
+    # test compute_ems cv
+    epochs = epochs['aud_r', 'vis_l']
+    epochs.equalize_event_counts(epochs.event_id)
+    if check_version('sklearn', '0.18'):
+        from sklearn.model_selection import StratifiedKFold
+        cv = StratifiedKFold()
+    else:
+        from sklearn.cross_validation import StratifiedKFold
+        cv = StratifiedKFold(len(epochs))
+    compute_ems(epochs, cv=cv)
+    compute_ems(epochs, cv=2)
+    compute_ems(epochs, cv=StratifiedKFold())
+    assert_raises(ValueError, compute_ems, epochs, cv='foo')
+    assert_raises(ValueError, compute_ems, epochs, cv=len(epochs) + 1)
```

Comment:
Yeah but it makes it immediately obvious that only 1 param is changing for each iteration. Not everything I say is about code golfing for fewer lines :)